microshift.sh

```diff
@@ -122,7 +122,7 @@ microshift_pkg_dir=$(mktemp -p /tmp -d tmp-rpmXXX)
 chmod 0755 ${microshift_pkg_dir}
 download_microshift_rpm ${microshift_pkg_dir}
 create_iso ${microshift_pkg_dir}
-sudo cp -Z microshift/_output/image-builder/microshift-installer-*.iso /var/lib/libvirt/images/microshift-installer.iso
+sudo cp -Z microshift/_output/image-builder/microshift-installer-*.iso /var/lib/libvirt/${SNC_PRODUCT_NAME}/microshift-installer.iso
```

Comment:
`microshift-installer.iso` might be leaked? `destroy_libvirt_resources` explicitly removes rhcos-live.iso, but not `microshift-installer.iso`.

Reply:
updated, now destroy_libvirt_resources need iso name as arg.